### ISSUE TITLE
Missing modules when installing mara

### ISSUE THREAD

**Issue by superzazu** [radiac/mara#2]
Hello,

Today I tried to run [this code](http://radiac.net/projects/mara/documentation/introduction/#a-minimal-service) (with Python 3), and I got the following error : 

```
Traceback (most recent call last):
  File "test1.py", line 1, in <module>
    from mara import Service, events
  File "/usr/local/lib/python3.5/site-packages/mara/__init__.py", line 3, in <module>
    from .service import Service
  File "/usr/local/lib/python3.5/site-packages/mara/service.py", line 15, in <module>
    from .angel import Process
  File "/usr/local/lib/python3.5/site-packages/mara/angel.py", line 17, in <module>
    from . import settings
ImportError: cannot import name 'settings'
```

Then, I realized that I don't have the folders `connection`, `contrib`, `events`, `settings`, `storage` and `timers` in the install path of mara (`/usr/local/lib/python3.5/site-packages/mara/`). I just did a `pip3 install mara`. Did I do something wrong ?

Best regards

**Comment by hagsteel** [comment]
Had the same issue, however installing with `pip install -e git+https://github.com/radiac/mara.git#egg=mara[full]` works for me.
See docs: http://radiac.net/projects/mara/documentation/installation/#installing-mara

Quite an old issue but figure I should put that here in case anyone else comes across this

**Comment by radiac** [comment]
There's an issue with how I packaged it for pypi where it's missing half the files. I thought I'd fixed it in 0.6.2, but still seems to be a problem. If you install it from github directly you'll get all the files; will put up a fixed version on pypi soon hopefully.

**Comment by radiac** [comment]
Mara v2 is now packaged correctly, v1 is no longer supported
